runtime/integration-tests/src/relaychain/statemine.rs

```diff
@@ -74,8 +77,14 @@ fn statemine_min_xcm_fee_matched() {
 }
 
 #[test]
-fn teleport_from_relay_chain() {
+fn teleport_between_relaychain_and_statemine_works() {
```

Comment:
Use cases between Kusama and statemine isn't in the scope of our integration tests.